scripts/check_archived_pv

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+usage()
+{
+cat << EOF
+usage: $0 <PV>
+
+Check on the status of a PV in the archiver, e.g. $0 GDET:FEE1:241:ENRC
+EOF
+}
+
+if [[ -z $1 ]]; then
+    usage
+    exit 1
+fi
+    
+if [[ ($1 == "--help") || ($1 == "-h") ]]; then
+    usage
+    exit 0
+fi
+
+result=$(curl -s "http://pscaa01.slac.stanford.edu:17665/mgmt/bpl/getPVStatus?pv=$1")
+echo
+echo $result | cut -d'{' -f2 | cut -d'}' -f1 | tr ',' '\n'
```

Comment:
Not suggesting a particular change, but you should know about the `jq` utility for processing/formatting json in the terminal:
```
zlentz@mfx-control:~$ curl -s "http://pscaa01.slac.stanford.edu:17665/mgmt/bpl/getPVStatus?pv=GDET:FEE1:241:ENRC" | jq
[
  {
    "lastRotateLogs": "Never",
    "appliance": "pscaa02",
    "pvName": "GDET:FEE1:241:ENRC",
    "pvNameOnly": "GDET:FEE1:241:ENRC",
    "connectionState": "true",
    "lastEvent": "Mar/02/2022 15:14:45 -08:00",
    "samplingPeriod": "1.0",
    "isMonitored": "false",
    "connectionLastRestablished": "Feb/09/2022 13:26:40 -08:00",
    "connectionFirstEstablished": "Sep/27/2021 09:59:53 -07:00",
    "connectionLossRegainCount": "11",
    "status": "Being archived"
  }
]
```

Reply:
More for awareness than as a suggestion

Reply:
Ah, nice; I remember someone mentioning this before, but it fell out of my brain at some point. Thankfully this JSON entry is pretty simple.